Default (OSX).sublime-keymap

```diff
@@ -7,7 +7,7 @@
 
   // Jump to a dependency
   {
-    "keys": ["super+alt+right"],
+    "keys": ["super+alt+down"],
```

Comment:
it's not the same now as linux but not likely too many will be jumping back and forth from osx to linux boxes and getting pissed, and I like "down" much better :-).   

A thought:  Is super alt left bound to anything in OSX.  You might want to change dependents binding to that so it's bilaterally pleasing :-).